public/app/types/explore.ts

```diff
@@ -270,3 +268,18 @@ export enum TABLE_RESULTS_STYLE {
 }
 export const TABLE_RESULTS_STYLES = [TABLE_RESULTS_STYLE.table, TABLE_RESULTS_STYLE.raw];
 export type TableResultsStyle = typeof TABLE_RESULTS_STYLES[number];
+
+export interface SuppQuery {
```

Comment:
`SuppQuery` -> `SupplementaryQuery`

Reply:
Fixed in https://github.com/grafana/grafana/pull/60998/commits/c4ba835aff6bb3e1f1e86f1e1ded853eda31ce9e